pkg/providers/cloudstack/cloudstack.go

```diff
@@ -476,10 +648,6 @@ func buildTemplateMapCP(clusterSpec *cluster.Spec, datacenterConfigSpec v1alpha1
 
 		// Add no-proxy defaults
 		noProxyList = append(noProxyList, common.NoProxyDefaults...)
-		cloudStackManagementApiEndpointHostname, err := getHostnameFromUrl(managementApiEndpoint)
-		if err == nil {
```

Comment:
after further discussion, we agreed to leave the no-proxy details in, and grab the managementApiEndpoint from the cluster spec, right?